pymtl3/dsl/test/sim_utils.py

```diff
@@ -497,3 +498,13 @@ def cleanup_signals( m ):
             cleanup_signals( obj )
 
   cleanup_signals( s )
+
+  def create_reset( top ):
+    def reset():
+      top.reset = Bits1(1)
+      top.tick()
+      top.tick()
```

Comment:
Can you remind me why we need to tick twice here? I remember previously it has something to do with top level ports. Do we still need to it now?

Reply:
Ideally I want to create special tick functions for this ... because we haven't solved the problem you are talking about. Basically you need to call eval_combinational between the top.reset and top.tick, but I don't create eval_combinational if I detect any CL primitive.